its/plugin/src/integrationTest/java/org/sonarsource/iac/PropertiesTest.java

```diff
@@ -126,11 +126,28 @@ void testDeprecatedArmActivationPropertyKey(String projectKey, Boolean armProper
     assertThat(getMeasureAsInt(projectKey, "ncloc")).isEqualTo(expectedNcloc);
   }
 
+  @ParameterizedTest
+  @CsvSource(delimiter = ';', value = {
+    "springConfigCustomPattern; application*.properties,config*.properties,*.yaml; 5; 13",
+    "springConfigCustomPropertiesPattern; application*.properties,config*.properties; 4; 4",
+    "springConfigExcludeBoot; config*.properties; 2; 2",
+    "springCustomYaml; *.yaml; 1; 9",
+  })
+  void testSpringConfigPatterns(String projectKey, String patterns, int expectedFiles, int expectedNCloc) {
+    var sonarScanner = getSonarScanner(projectKey, BASE_DIRECTORY + "patterns", "java", "springconfig-its");
+    sonarScanner.setProperty("sonar.java.springconfig.file.patterns", patterns);
+
+    ORCHESTRATOR.executeBuild(sonarScanner);
+
+    assertThat(getMeasureAsInt(projectKey, "files")).isEqualTo(expectedFiles);
+    assertThat(getMeasureAsInt(projectKey, "ncloc")).isEqualTo(expectedNCloc);
+  }
+
```

Comment:
Please also add a test for `sonar.java.springconfig.activate`.

Reply:
We don't have tests for this property for other sensors, only for ARM, where we have two properties. So I thought it's not required for the new one